src/NServiceBus.Core/Serializers/Json/JsonMessageSerializer.cs

```diff
@@ -12,12 +12,11 @@ namespace NServiceBus.Serializers.Json
     using Newtonsoft.Json.Converters;
     using NServiceBus.Serialization;
 
-    /// <summary>
-    /// JSON message serializer.
-    /// </summary>
-    public class JsonMessageSerializer : IMessageSerializer
+    class JsonMessageSerializer : IMessageSerializer
```

Comment:
That has always been a smell IMO. How we craft the json manually or include simplejson or a similar serializer in the plugins?